zulipterminal/model.py

```diff
@@ -1318,20 +1319,7 @@ def notify_user(self, message: Message) -> str:
                 text = content
             else:
                 soup = BeautifulSoup(content, "lxml")
```

Comment:
This could nicely go into the notification_formatter function instead; leaving it here exposes the details of how we're doing it, rather than just passing the message content to the function.